### PR TITLE
[GENERAL UPDATE] Add variables $VERSION to agility the installing and updating process of the services

### PR DESCRIPTION
**Pull request by Swissnode**
#### What

Added two variables $VERSION so that users upgrading to a version that is not already mirrored in the documentation only have to change the VERSION variable once and can merely copy-paste the other lines

### Why

Helps users to upgrade quicker. Very simple change too!

#### How

Code change to add the two variables mentioned above

#### Scope

- [ ] significant change to core configuration
- [ ] independent bonus guide
- [ ] simple bug fix

Fixes # (link issue)
N/A - not a fix

#### Test & maintenance

Can be tested by merely updating a future lnd upgrade using it

### CODE REVIEW

File: guide/lightning/lightning-client.md [hunk 1]
```diff
@@ -32,19 +32,21 @@ We'll download, verify and install LND.
 
   ```sh
   $ cd /tmp
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/lnd-linux-arm64-v0.15.5-beta.tar.gz
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-v0.15.5-beta.txt
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-roasbeef-v0.15.5-beta.sig
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-roasbeef-v0.15.5-beta.sig.ots
+  $ VERSION="v0.15.5-beta"
```
[the remark this answers is not in the document]
Comment (kristapsk) [reply guide/lightning/lightning-client.md:35]: It creates variable and assigns value to it. Works the same way in interactive bash shell and bash scripts.

**Comment by kristapsk** [comment]
Concept ACK. Will be able to look at it more carefully and test in a few days.

File: guide/lightning/lightning-client.md [hunk 2]
```diff
@@ -63,7 +65,7 @@ Now that we've verified the integrity of the downloaded binary, we need to check
 * Verify the signature of the text file containing the checksums for the application
 
   ```sh
-  $ gpg --verify manifest-roasbeef-v0.15.5-beta.sig manifest-v0.15.5-beta.txt
+  $ gpg --verify manifest-roasbeef-$VERSION.sig manifest-v0.15.5-beta.txt
```
Comment (vicariousdrama) [inline guide/lightning/lightning-client.md:68]: Need the $VERSION reference to be used in the manifest filename as well.

File: guide/lightning/lightning-client.md [hunk 3]
```diff
@@ -32,19 +32,21 @@ We'll download, verify and install LND.
 
   ```sh
   $ cd /tmp
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/lnd-linux-arm64-v0.15.5-beta.tar.gz
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-v0.15.5-beta.txt
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-roasbeef-v0.15.5-beta.sig
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-roasbeef-v0.15.5-beta.sig.ots
+  $ VERSION="v0.15.5-beta"
+  $ DISTRO="lnd-linux-arm64"
+  $ wget https://github.com/lightningnetwork/lnd/releases/download/$VERSION/$DISTRO-$VERSION.tar.gz
```
Comment (kristapsk) [inline guide/lightning/lightning-client.md:37]: I think this makes more sense.
```suggestion
  $ DISTRO="linux-arm64"
  $ wget https://github.com/lightningnetwork/lnd/releases/download/$VERSION/lnd-$DISTRO-$VERSION.tar.gz
```

File: guide/lightning/lightning-client.md [hunk 4]
```diff
@@ -32,19 +32,21 @@ We'll download, verify and install LND.
...
 
 ### Checksum check
 
 * Verify the signed checksum against the actual checksum of your download
 
   ```sh
-  $ sha256sum --check manifest-v0.15.5-beta.txt --ignore-missing
-  > lnd-linux-arm64-v0.15.5-beta.tar.gz: OK
+  $ sha256sum --check manifest-$VERSION.txt --ignore-missing
+  > lnd-linux-arm64-$VERSION.tar.gz: OK
```
Comment (kristapsk) [inline guide/lightning/lightning-client.md:49]: You will not have `$VERSION` but actual version in output here.
```suggestion
  > lnd-linux-arm64-v0.15.5-beta.tar.gz: OK
```

**Comment by twofaktor** [comment]
Good add-on! This will go to agility the updating process of the services a lot. BTW: Could apply this to other services installation e.g to the Bitcoin client as well?

File: guide/lightning/lightning-client.md (hunk 1 above)
Comment (twofaktor) [inline guide/lightning/lightning-client.md:35]: Should we call this `LNDVERSION="v0.15.5-beta"` to can use different variables for each service in the future?

File: guide/lightning/lightning-client.md [hunk 5]
```diff
@@ -32,19 +32,21 @@ We'll download, verify and install LND.
 
   ```sh
   $ cd /tmp
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/lnd-linux-arm64-v0.15.5-beta.tar.gz
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-v0.15.5-beta.txt
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-roasbeef-v0.15.5-beta.sig
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-roasbeef-v0.15.5-beta.sig.ots
+  $ VERSION="v0.15.5-beta"
+  $ DISTRO="lnd-linux-arm64"
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:36]: Should we call this `LNDDISTRO="v0.15.5-beta"` to can use different variables for each service in the future? 

i.e: BTCVERSION / BTCDISTRO

**Review by twofaktor — CHANGES_REQUESTED** [review]

**Comment by Mynima** [comment]
> Good add-on! This will go to agility the updating process of the services a lot. BTW: Could apply this to other services installation e.g to the Bitcoin client as well?

yeah I was wondering the same thing. It would be great to add this same methodology with the other sections. Def think Bitcoin Client is a good place to start.

File: guide/lightning/lightning-client.md (hunk 4 above)
Comment (Mynima) [reply guide/lightning/lightning-client.md:49]: @Swissnode this minor update still outstanding

File: guide/lightning/lightning-client.md (hunk 2 above)
Comment (Mynima) [reply guide/lightning/lightning-client.md:68]: @Swissnode this minor update still outstanding

**Review by Mynima — COMMENTED** [review]
ACK Couple of minor comments to be resolved. Have tested throughout without issue.

File: guide/lightning/lightning-client.md (hunk 3 above)
Comment (twofaktor) [reply guide/lightning/lightning-client.md:37]: DISTRO variable is not necessary, RaspiBolt will be always for arm64

**Comment by twofaktor** [comment]
> $ BVERSION="24.0.1" 

Why "BVERSION" and not "VERSION" ??

**Comment by Mynima** [comment]
> > $ BVERSION="24.0.1"
> 
> Why "BVERSION" and not "VERSION" ??

Absolutely no reason to not be VERSION and probably should be for consistency.

**Comment by twofaktor** [comment]
> > > $ BVERSION="24.0.1"
> > 
> > 
> > Why "BVERSION" and not "VERSION" ??
> 
> Absolutely no reason to not be VERSION and probably should be for consistency.

Environment variables are temporal definitions, these only keep in an current session

**Comment by twofaktor** [comment]
@Swissnode are you available to do modifications? do you prefer we close this PR and I open a new one so I can take charge

**Comment by Mynima** [comment]
yo @Swissnode would you have time to update this one for the new LND release [lnd v0.16.1-beta](https://github.com/lightningnetwork/lnd/releases/tag/v0.16.1-beta), I'm happy to do the testing when I upgrade at the weekend. Be great to get this PR closed out.

File: guide/lightning/lightning-client.md (hunk 2 above)
Comment (Swissnode) [reply guide/lightning/lightning-client.md:68]: oh shit sorry, didn't see that, as I hardly ever log on to Github.

Will get on it...

File: guide/lightning/lightning-client.md (hunk 4 above)
Comment (Swissnode) [reply guide/lightning/lightning-client.md:49]: yeah i didn't realise... on it... (my first ever PR)

**Comment by Swissnode** [comment]
Hey guys,

Really sorry for the delay - I will try and get on my account more often.

I implemented the changes in my fork. Can you have a look?

@twofaktor thanks

**Comment by Mynima** [comment]
> Could we add Bitcoin Core and other possible services to this PR too?
> 
> I already integrated that on MiniBolt guide, we could get references of there if you agree

Yeah I think this could be a good idea to get them done at the same time. Will you be able to do that @Swissnode?

**Comment by Swissnode** [comment]
> > Could we add Bitcoin Core and other possible services to this PR too?
> > I already integrated that on MiniBolt guide, we could get references of there if you agree
> 
> Yeah I think this could be a good idea to get them done at the same time. Will you be able to do that @Swissnode?

Yeah I guess I can do that!!

Excuse my Github noobishness on this front. I assume I go into my own SwissNode repository and do the changes there correct? Do I somehow have to tag WHICH files are included or will Github assume that ALL changes i make to my repository will be taken into account for this PR?

**Comment by Swissnode** [comment]
ok @Mynima and @twofaktor 

I see it does pick up all the files I changed so I am now fairly sure I got the changes correctly made. Wanna review and pull them in?

Updated bitcoin-client.md and lightning-client.md

File: guide/bitcoin/bitcoin-client.md [hunk 6]
```diff
@@ -44,14 +44,18 @@ This is a precaution to make sure that this is an official release and not a mal
 * Get the latest download links at [bitcoincore.org/en/download](https://bitcoincore.org/en/download){:target="_blank"} (ARM Linux 64 bit), they change with each update.
 
   ```sh
+  # set up some version variables for easier maintenance later on
+  $ VERSION_CORE="bitcoin-core-25.0"
+  $ VERSION="bitcoin-25.0"
+  
   # download Bitcoin Core binary
-  $ wget https://bitcoincore.org/bin/bitcoin-core-23.0/bitcoin-23.0-aarch64-linux-gnu.tar.gz
+  $ wget https://bitcoincore.org/bin/$VERSION_CORE/$VERSION-aarch64-linux-gnu.tar.gz
```
Comment (twofaktor) [inline guide/bitcoin/bitcoin-client.md:52]:
```suggestion
  $ wget https://bitcoincore.org/bin/bitcoin-core-$VERSION/bitcoin-$VERSION-aarch64-linux-gnu.tar.gz
```

File: guide/bitcoin/bitcoin-client.md [hunk 7]
```diff
@@ -44,14 +44,18 @@ This is a precaution to make sure that this is an official release and not a mal
 * Get the latest download links at [bitcoincore.org/en/download](https://bitcoincore.org/en/download){:target="_blank"} (ARM Linux 64 bit), they change with each update.
 
   ```sh
+  # set up some version variables for easier maintenance later on
+  $ VERSION_CORE="bitcoin-core-25.0"
+  $ VERSION="bitcoin-25.0"
```
Comment (twofaktor) [inline guide/bitcoin/bitcoin-client.md:49]:
```suggestion
  $ VERSION="25.0"
```

File: guide/bitcoin/bitcoin-client.md [hunk 8]
```diff
@@ -44,14 +44,18 @@ This is a precaution to make sure that this is an official release and not a mal
...
+  
   # download Bitcoin Core binary
-  $ wget https://bitcoincore.org/bin/bitcoin-core-23.0/bitcoin-23.0-aarch64-linux-gnu.tar.gz
+  $ wget https://bitcoincore.org/bin/$VERSION_CORE/$VERSION-aarch64-linux-gnu.tar.gz
 
   # download the list of cryptographic checksum
-  $ wget https://bitcoincore.org/bin/bitcoin-core-23.0/SHA256SUMS
+  $ wget https://bitcoincore.org/bin/$VERSION_CORE/SHA256SUMS
```
Comment (twofaktor) [inline guide/bitcoin/bitcoin-client.md:55]:
```suggestion
  $ wget https://bitcoincore.org/bin/bitcoin-core-$VERSION/SHA256SUMS
```

File: guide/bitcoin/bitcoin-client.md [hunk 9]
```diff
@@ -44,14 +44,18 @@ This is a precaution to make sure that this is an official release and not a mal
...
 
   # download the signatures attesting to validity of the checksums
-  $ wget https://bitcoincore.org/bin/bitcoin-core-23.0/SHA256SUMS.asc
+  $ wget https://bitcoincore.org/bin/$VERSION_CORE/SHA256SUMS.asc
```
Comment (twofaktor) [inline guide/bitcoin/bitcoin-client.md:58]:
```suggestion
  $ wget https://bitcoincore.org/bin/bitcoin-core-$VERSION/SHA256SUMS.asc
```

File: guide/bitcoin/bitcoin-client.md [hunk 10]
```diff
@@ -91,8 +95,8 @@ This is a precaution to make sure that this is an official release and not a mal
 ### Timestamp check
 
 * The binary checksum file is timestamped on the Bitcoin blockchain via the [OpenTimestamps protocol](https://opentimestamps.org/){:target="_blank"}, proving that the file existed prior to some point in time. Let's verify this timestamp. On your local computer, download the checksums file and its timestamp proof:
-  *  https://bitcoincore.org/bin/bitcoin-core-23.0/SHA256SUMS.ots
-  *  https://bitcoincore.org/bin/bitcoin-core-23.0/SHA256SUMS
+  *  https://bitcoincore.org/bin/bitcoin-25.0/SHA256SUMS.ots
```
Comment (twofaktor) [inline guide/bitcoin/bitcoin-client.md:98]:
```suggestion
  *  https://bitcoincore.org/bin/bitcoin-core-25.0/SHA256SUMS.ots
```

File: guide/bitcoin/bitcoin-client.md [hunk 11]
```diff
@@ -91,8 +95,8 @@ This is a precaution to make sure that this is an official release and not a mal
...
+  *  https://bitcoincore.org/bin/bitcoin-25.0/SHA256SUMS
```
Comment (twofaktor) [inline guide/bitcoin/bitcoin-client.md:99]:
```suggestion
  *  https://bitcoincore.org/bin/bitcoin-core-25.0/SHA256SUMS
```

File: guide/bitcoin/bitcoin-client.md [hunk 12]
```diff
@@ -105,10 +109,10 @@ This is a precaution to make sure that this is an official release and not a mal
 * If you're satisfied with the checkum, signature and timestamp checks, extract the Bitcoin Core binaries, install them and check the version.
 
   ```sh
-  $ tar -xvf bitcoin-23.0-aarch64-linux-gnu.tar.gz
-  $ sudo install -m 0755 -o root -g root -t /usr/local/bin bitcoin-23.0/bin/*
+  $ tar -xvf $VERSION-aarch64-linux-gnu.tar.gz
```
Comment (twofaktor) [inline guide/bitcoin/bitcoin-client.md:112]:
```suggestion
  $ tar -xvf bitcoin-$VERSION-aarch64-linux-gnu.tar.gz
```

File: guide/bitcoin/bitcoin-client.md [hunk 13]
```diff
@@ -105,10 +109,10 @@ This is a precaution to make sure that this is an official release and not a mal
...
+  $ sudo install -m 0755 -o root -g root -t /usr/local/bin $VERSION/bin/*
```
Comment (twofaktor) [inline guide/bitcoin/bitcoin-client.md:113]:
```suggestion
  $ sudo install -m 0755 -o root -g root -t /usr/local/bin bitcoin-$VERSION/bin/*
```

File: guide/lightning/lightning-client.md [hunk 14]
```diff
@@ -32,19 +32,20 @@ We'll download, verify and install LND.
 
   ```sh
   $ cd /tmp
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/lnd-linux-arm64-v0.15.5-beta.tar.gz
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-v0.15.5-beta.txt
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-roasbeef-v0.15.5-beta.sig
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.15.5-beta/manifest-roasbeef-v0.15.5-beta.sig.ots
+  $ VERSION="v0.15.5-beta"
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:35]:
```suggestion
  $ VERSION="0.16.3"
```

File: guide/lightning/lightning-client.md [hunk 15]
```diff
@@ -32,19 +32,20 @@ We'll download, verify and install LND.
...
+  $ wget https://github.com/lightningnetwork/lnd/releases/download/$VERSION/lnd-linux-arm64-$VERSION.tar.gz
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:36]:
```suggestion
  $ wget https://github.com/lightningnetwork/lnd/releases/download/v$VERSION-beta/lnd-linux-arm64-v$VERSION-beta.tar.gz
```

File: guide/lightning/lightning-client.md [hunk 16]
```diff
@@ -32,19 +32,20 @@ We'll download, verify and install LND.
...
+  $ wget https://github.com/lightningnetwork/lnd/releases/download/$VERSION/manifest-$VERSION.txt
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:37]:
```suggestion
  $ wget https://github.com/lightningnetwork/lnd/releases/download/v$VERSION-beta/manifest-v$VERSION-beta.txt
```

File: guide/lightning/lightning-client.md [hunk 17]
```diff
@@ -32,19 +32,20 @@ We'll download, verify and install LND.
...
+  $ wget https://github.com/lightningnetwork/lnd/releases/download/$VERSION/manifest-roasbeef-$VERSION.sig
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:38]:
```suggestion
  $ wget https://github.com/lightningnetwork/lnd/releases/download/v$VERSION-beta/manifest-roasbeef-v$VERSION-beta.sig
```

File: guide/lightning/lightning-client.md [hunk 18]
```diff
@@ -32,19 +32,20 @@ We'll download, verify and install LND.
...
+  $ wget https://github.com/lightningnetwork/lnd/releases/download/$VERSION/manifest-roasbeef-$VERSION.sig.ots
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:39]:
```suggestion
  $ wget https://github.com/lightningnetwork/lnd/releases/download/v$VERSION-beta/manifest-roasbeef-v$VERSION-beta.sig.ots
```

File: guide/lightning/lightning-client.md [hunk 19]
```diff
@@ -32,19 +32,20 @@ We'll download, verify and install LND.
...
   ```
 
 ### Checksum check
 
 * Verify the signed checksum against the actual checksum of your download
 
   ```sh
-  $ sha256sum --check manifest-v0.15.5-beta.txt --ignore-missing
-  > lnd-linux-arm64-v0.15.5-beta.tar.gz: OK
+  $ sha256sum --check manifest-$VERSION.txt --ignore-missing
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:47]:
```suggestion
  $ sha256sum --check manifest-v$VERSION-beta.txt --ignore-missing
```

File: guide/lightning/lightning-client.md [hunk 20]
```diff
@@ -32,19 +32,20 @@ We'll download, verify and install LND.
...
+  > lnd-linux-arm64-$VERSION.tar.gz: OK
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:48]:
```suggestion
  > lnd-linux-arm64-v$VERSION-beta.tar.gz: OK
```

File: guide/lightning/lightning-client.md [hunk 21]
```diff
@@ -63,7 +64,7 @@ Now that we've verified the integrity of the downloaded binary, we need to check
 * Verify the signature of the text file containing the checksums for the application
 
   ```sh
-  $ gpg --verify manifest-roasbeef-v0.15.5-beta.sig manifest-v0.15.5-beta.txt
+  $ gpg --verify manifest-roasbeef-$VERSION.sig manifest-$VERSION.txt
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:67]:
```suggestion
  $ gpg --verify manifest-roasbeef-v$VERSION-beta.sig manifest-v$VERSION-beta.txt
```

File: guide/lightning/lightning-client.md [hunk 22]
```diff
@@ -77,7 +78,7 @@ We can also check that the manifest file was in existence around the time of the
 * Let's verify the timestamp of the file matches the release date.
 
   ```sh
-  $ ots verify manifest-roasbeef-v0.15.5-beta.sig.ots -f manifest-roasbeef-v0.15.5-beta.sig
+  $ ots verify manifest-roasbeef-$VERSION.sig.ots -f manifest-roasbeef-$VERSION.sig
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:81]:
```suggestion
  $ ots --no-cache verify manifest-roasbeef-v$VERSION-beta.sig.ots -f manifest-roasbeef-v$VERSION-beta.sig
```

File: guide/lightning/lightning-client.md [hunk 23]
```diff
@@ -91,8 +92,8 @@ Having verified the integrity and authenticity of the release binary, we can saf
 * Install LND
 
   ```sh
-  $ tar -xzf lnd-linux-arm64-v0.15.5-beta.tar.gz
-  $ sudo install -m 0755 -o root -g root -t /usr/local/bin lnd-linux-arm64-v0.15.5-beta/*
+  $ tar -xzf lnd-linux-arm64-$VERSION.tar.gz
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:95]:
```suggestion
  $ tar -xvf lnd-linux-arm64-v$VERSION-beta.tar.gz
```

File: guide/lightning/lightning-client.md [hunk 24]
```diff
@@ -91,8 +92,8 @@ Having verified the integrity and authenticity of the release binary, we can saf
...
+  $ sudo install -m 0755 -o root -g root -t /usr/local/bin lnd-linux-arm64-$VERSION/*
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:96]:
```suggestion
  $ sudo install -m 0755 -o root -g root -t /usr/local/bin lnd-linux-arm64-v$VERSION-beta/*
```

File: guide/lightning/lightning-client.md [hunk 25]
```diff
@@ -91,8 +92,8 @@ Having verified the integrity and authenticity of the release binary, we can saf
...
   $ lnd --version
   > lnd version 0.15.5-beta commit=v0.15.5-beta
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:98]:
```suggestion
  > lnd version $VERSION-beta commit=v$VERSION-beta
```

**Review by twofaktor — CHANGES_REQUESTED** [review]

File: guide/lightning/lightning-client.md (hunk 4 above)
Comment (Swissnode) [reply guide/lightning/lightning-client.md:48]: I think this change is taken care of in a later suggestion... (was getting conflicting lines) - i think we can ignore this suggestion from December...

**Comment by Swissnode** [comment]
@twofaktor thanks for the help, I think it was more useful for my Github skills than the changes themselves. You are right, was better to have just the numbers in the version and not the word "version". I checked your changes and they all checked out fine...

Think it's good to go - am ignoring the December changes as I think there are superceded

**Comment by Swissnode** [comment]
I seem to have some "branch conflicts" but tbh i don't understand why it says this. Can you have a look?

**Review by twofaktor — APPROVED** [review]
tACK!

**Review by Mynima — CHANGES_REQUESTED** [review]
Please update the follow lines of code in the "Future Update for Bitcoin Core" section.

Before this bit add ' VERSION = "25.0"
image

Then correct the above 4 lines to use "$VERSION"

image

Then correct the above 2 lines to use "$VERSION"

**Comment by Mynima** [comment]
> ok @Mynima and @twofaktor
> 
> I see it does pick up all the files I changed so I am now fairly sure I got the changes correctly made. Wanna review and pull them in?
> 
> Updated bitcoin-client.md and lightning-client.md

Some minor comments added. Thanks for all the work on this! Last push and we should get it across the line!!

File: guide/lightning/lightning-client.md [hunk 26]
```diff
@@ -32,18 +32,19 @@ We'll download, verify and install LND.
 
   ```sh
   $ cd /tmp
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.16.3-beta/lnd-linux-arm64-v0.16.3-beta.tar.gz
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.16.3-beta/manifest-v0.16.3-beta.txt
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.16.3-beta/manifest-roasbeef-v0.16.3-beta.sig
-  $ wget https://github.com/lightningnetwork/lnd/releases/download/v0.16.3-beta/manifest-roasbeef-v0.16.3-beta.sig.ots
+  $ VERSION="0.16.3"
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:35]:
```suggestion
  $ VERSION="0.16.3"
```
Add before this line:
$ VERSION="25.0"

@Swissnode

**Review by twofaktor — CHANGES_REQUESTED** [review]
Add before this line: https://github.com/raspibolt/raspibolt/pull/1200/files#diff-1dad6164a6192352aaa1591ee62dc908f604c7df9cdae29a83ba0a0585b91539L528

`$ VERSION=25.0`

**Comment by Mynima** [comment]
> Add instead of this line: https://github.com/raspibolt/raspibolt/pull/1200/files#diff-1dad6164a6192352aaa1591ee62dc908f604c7df9cdae29a83ba0a0585b91539L528
> 
> `$ VERSION=25.0`
> 
> @Swissnode, please

Both should work tbh

**Review by Mynima — APPROVED** [review]
tACK - LND

Bitcoin guide just needs a few bits added to the "Future Update for Bitcoin Core" section

**Comment by twofaktor** [comment]
All ready, does someone want to do a final review?

**Review by Mynima — APPROVED** [review]
LGTM 
Bitcoin Core - utACK
LND - tACK

**Review by twofaktor — APPROVED** [review]

**Review by twofaktor — APPROVED** [review]
OK! All ready! Let's go to the master! 🚀

**Comment by Swissnode** [comment]
exciting! it might be rather banal but it was my first ever PR, so I won't forget it! :D

**Pull request merged**